index.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This is an API call to the pokeAPI and pulls all of the data from 151 pokemon and puts them in an array
+ *
+ *
+ * @result this is a json string of all of the information of all of the 151 pokemon called
```

Comment:
This is not a json string with all the results in. It is an array of json string objects with the information in it.